src/base/numbers.c

```diff
@@ -74,3 +74,42 @@ int _getHexString(char *buffer, unsigned long hexnumber, bool use_capital) {
     }
     return size;
 }
+
+// Unsigned
+int itoa(char *buffer, unsigned long number, int base, bool use_capital) {
+	if (base < 1 || base > 36) return 0;
+	char *pointer, *pointerbase;
+	int mod;
+	int size = 0;
+	char letter_base = 'a';
+	if (use_capital) {
+		letter_base = 'A';
+	}
+	pointer = buffer;
+	pointerbase = buffer;
+	if(number == 0) {
+		*pointer = '0';
+		return 1;
+	}
+	while (number > 0) {
+		mod = number % base;
+		if (mod < 10) {
+			*pointer++ = '0' + mod;
+		} else {
+			*pointer++ = letter_base + mod - 10;
+		}
+		number = number / base;
+		size++;
+	}
+	*pointer--=0;
```

Comment:
The function changes its intent here, would be good to add a new blank line to show separation again.
Also this line is a bit dense, try adding some spaces: `*pointer-- = 0;`